src/amo/components/App/styles.scss

```diff
@@ -44,6 +44,11 @@ body {
   padding: 0 0 $padding-page;
 }
 
+.App-content > div {
```

Comment:
Thinking a dedicated wrapper would be a plan, to save the possibility of this being brittle in the future.

Reply:
Yeah this should be changed I'd say before we merge, it should be easy, right?